app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -14,7 +14,7 @@ external_content:
 promo_content:
     - content/train-to-be-a-teacher/promos/adviser-promo-degree
 navigation: 20.05
-navigation_title: If you have or are studying for a degree
+navigation_title: Train to teach if you have or are studying for a degree
```

Comment:
Title on the tile and h1 don't match. 
Train to teach if you have or are studying for a degree - H1 on tile
Train to teach if you have a degree or are studying for one - Page title